oap-server-bom/pom.xml

```diff
@@ -73,7 +73,7 @@
         <awaitility.version>3.0.0</awaitility.version>
         <httpcore.version>4.4.13</httpcore.version>
         <commons-compress.version>1.21</commons-compress.version>
-        <banyandb-java-client.version>0.2.0</banyandb-java-client.version>
+        <banyandb-java-client.version>0.3.0-SNAPSHOT</banyandb-java-client.version>
```

Comment:
@lujiajing1126 I think you are able to initialize the 0.2.1 release.

Reply:
Sure. We could cherry-pick the last commit and release a patch version.